src/reflect/scala/reflect/internal/Mirrors.scala

```diff
@@ -197,29 +197,35 @@ trait Mirrors extends api.Mirrors {
         case _                                                   => MissingRequirementError.notFound("package " + fullname)
       }
 
-    @deprecated("Use overload that accepts a String.", "2.13.0")
+    @deprecated("use overload that accepts a String.", since = "2.13.0")
     def getPackage(fullname: TermName): ModuleSymbol =
       getPackage(fullname.toString)
     def getPackage(fullname: String): ModuleSymbol =
       ensurePackageSymbol(fullname, getModuleOrClass(fullname, newTermName(_)), allowModules = true)
 
+    @deprecated("use overload that accepts a String.", since = "2.12.11")
```

Comment:
Now I don't remember if it's period or no period.

The reporter should add `!` if no punctuation.
```
use overload that accepts a String!
```

Reply:
The line between news and opinion is dangerously blurred these days; what say you we blur the line between reporter and editor and provide an `EditorializingReporter`, for those tired of our bland error messages? We say `side effecting nullary methods are discouraged`, for instance, why not `suggest defining as def foo: IO[Unit] instead`?